pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -1033,7 +1037,7 @@ func (a byPVCSize) Less(i, j int) bool {
 }
 
 // isCSIMigrationOnForPlugin checks if CSI migration is enabled for a given plugin.
-func isCSIMigrationOnForPlugin(pluginName string) bool {
+func isCSIMigrationOnForPlugin(enableCSIMigrationPortworx bool, pluginName string) bool {
```

Comment:
Would it better that we adjust the order of args
```suggestion
func isCSIMigrationOnForPlugin(pluginName string, enableCSIMigrationPortworx bool) bool {
```

Reply:
done